pom.xml

```diff
@@ -43,29 +43,20 @@
         </developer>
     </developers>
 
-    <licenses>
-        <license>
-            <name>MIT License</name>
-            <url>http://opensource.org/licenses/MIT</url>
-        </license>
-    </licenses>
-    <repositories>
-        <repository>
-            <id>repo.jenkins-ci.org</id>
-            <url>http://repo.jenkins-ci.org/public/</url>
-        </repository>
-    </repositories>
-    <pluginRepositories>
-        <pluginRepository>
-            <id>repo.jenkins-ci.org</id>
-            <url>http://repo.jenkins-ci.org/public/</url>
-        </pluginRepository>
-    </pluginRepositories>
+    <scm>
+        <connection>scm:git:ssh://github.com/jenkinsci/statistics-gatherer-plugin.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/jenkinsci/statistics-gatherer-plugin.git</developerConnection>
+        <url>https://github.com/jenkinsci/statistics-gatherer-plugin</url>
+        <tag>HEAD</tag>
+    </scm>
+
     <properties>
         <jenkins.version>2.60.1</jenkins.version>
         <java.level>8</java.level>
         <powermock.version>1.7.4</powermock.version>
+        <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
I believe we should fail on errors, is there any reason for not doing that?

Reply:
Findbug was completely skipped before. I enabled it and it shows 33 issues. I'll not fix all of them. This option can only be enabled if they are fixed.